scipy/stats/tests/test_stats.py

```diff
@@ -925,6 +925,56 @@ def test_index_error(self):
             0.0007835762419683435])
         assert_raises(ValueError, stats.spearmanr, x, y, axis=2)
 
+    def test_alternative(self):
+        # Test alternative parameter
+        assert_raises(ValueError, stats.spearmanr, X, X,
+                      alternative='error')
+
+        # Simple test - Based on the above ``test_spearmanr_vs_r``
+        x1 = [1, 2, 3, 4, 5]
+        x2 = [5, 6, 7, 8, 7]
+        expected = (0.82078268166812329, 0.088587005313543798)
+
+        res = stats.spearmanr(x1, x2, alternative="less")
+        assert_approx_equal(res[0], expected[0])
+        assert_approx_equal(res[1], 1 - (expected[1] / 2))
```

Comment:
These are correct. You can test against R:
```
options(digits=16)
cor.test(c(1, 2, 3, 4, 5), c(5, 6, 7, 8, 7), alternative='less', method='spearman')
```
In this case, "greater" and "less" can be thought of as the true rho is GT/LT 0.